ui/desktop/electron-app/config/cli.js

```diff
@@ -1,25 +1,39 @@
 const fs = require('fs');
+const os = require('os');
 const path = require('path');
 const https = require('https');
 const decompress = require('decompress');
-const os = require('os');
+const { isMac, isWindows } = require('../src/helpers/platform.js');
 
-const artifactVersion = '0.2.2';
+const artifactVersion = '0.2.3';
 const artifactDestination = path.resolve(__dirname, '..', 'cli');
 
-const downloadArtifact = version => {
-  const url = `https://releases.hashicorp.com/boundary/${version}/boundary_${version}_darwin_amd64.zip`;
+const downloadArtifact = (version) => {
+  const archivePlatform = {};
+  // TODO: Support all available build machines
+  if (isMac()) {
+    archivePlatform.name = 'darwin';
+    archivePlatform.arch = 'amd64';
+  }
+
+  if (isWindows()) {
+    archivePlatform.name = 'windows';
+    archivePlatform.arch = 'amd64';
```

Comment:
Does `arch` always return `amd64`? do we have no cases where is `x86`?

Reply:
For windows, cli for is supported for amd64 and 386. However, desktop is aimed for Windows 10 and above, thus hardcoded to 'amd64'.

Reply:
Ahh gotcha! for now more than enough, I was using these questions as a shortcuts to learn too.